addOns/reports/src/main/zapHomeFiles/reports/sarif-json/report.json

```diff
@@ -0,0 +1,140 @@
+               },
+               "taxa": [[#th:block th:each="sarifTaxa, sarifTaxaState: ${sarifTaxonomy.taxa}"]
+                  {
+                     "guid": "[(${sarifTaxa.guid})]",
+                     "helpUri": "[(${sarifTaxa.helpUri})]",
+                     "id": "[(${sarifTaxa.id})]"
+                  }[#th:block th:if="${! sarifTaxaState.last}"],[/th:block]
+               [/th:block]],
+               "version": "4.4"
+            }[#th:block th:if="${! sarifTaxonomyState.last}"],[/th:block]
+         [/th:block]],
+         "tool": {
+            "driver": {
+               "guid": "[(${sarifSupport.components.owaspZap.guid})]",
+               "informationUri": "[(${sarifSupport.components.owaspZap.informationUri})]",
+               "name": "[(${sarifSupport.components.owaspZap.name})]",
+               "rules": [ [#th:block th:each="sarifRule, sarifRuleState: ${sarifSupport.rules}"]
+                  {
+                     "id": "[(${sarifRule.id})]",
+                     "defaultConfiguration": {
+                        "level": "[(${sarifRule.defaultLevel.value})]"
+                     },
+                     "fullDescription": {
+                        "text": [[(${sarifRule.fullDescription})]]
+                     },
+                     "name": "[(${sarifRule.name})]",
+                     "properties": {
+                            "references": [[#th:block th:each="sarifReference, sarifReferenceState: ${sarifRule.properties.references}"]
+                                "[(${sarifReference})]"[#th:block th:if="${! sarifReferenceState.last}"],[/th:block]
+                            [/th:block]],
+                            "solution": {
+                                "text": [[(${sarifRule.properties.solution.text})]]
+                            },
+                            "confidence": "[(${sarifRule.properties.confidence})]"
+                     },
+                     "relationships": [ [#th:block th:each="relationShip, relationShipState: ${sarifRule.relationShips}"]
+                        {
+                           "kinds": [
+                              "superset"
+                           ],
+                           "target": {
+                              "guid":"[(${relationShip.target.guid})]",
+                              "id": "[(${relationShip.target.id})]",
+                              "toolComponent": {
+                                 "guid": "[(${relationShip.target.toolComponent.guid})]",
+                                 "name": "[(${relationShip.target.toolComponent.name})]"
+                              }
+                           }
+                        }[#th:block th:if="${! relationShipState.last}"],[/th:block]
+                        [/th:block]
+                     ],
+                     "shortDescription": {
+                        "text": "[(${sarifRule.shortDescription})]"
```

Comment:
Same here.

Reply:
I will extend the report integration test with problematic chars inside. Will also try to change all occurrences of "[(" to "[[" where possible.